### DISCUSSION TITLE
Double refresh indicator in notification list

### DISCUSSION THREAD

**Discussion by DevDengChao** [community/community#64191] [Mobile] [answered]
As you can see.

![Screenshot_2023-08-18-17-45-06-980_com.github.android.jpg](https://github.com/community/community/assets/16363180/0abc8968-5677-4a65-9bcc-d23f2a357719)

**Comment by Iron-Ham** [comment]
Thank you for reporting this @DevDengChao! It's your feedback that helps us make GitHub Mobile a better experience. 

Could you provide a bit more information:
Does this occur only when there are few notifications in the notification list, or does it always occur when you trigger a pull-t-refresh?

I'll file this as a bug so that we can prioritize this internally. Thank you!

**Reply by DevDengChao** [reply]
There are always two indicators.

**Reply by Iron-Ham** [reply, marked as answer]
@DevDengChao I'm happy to report that we've merged a fix to this issue. It should be released in next week's beta (August 29), and in production the following week (September 5).

**Comment by Binary-Ninja-007** [comment]
Good.
